app/src/main/java/com/example/tmdb/modules/Movie.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.tmdb.modules
+
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.ui.graphics.painter.Painter
+
+data class Movie(
```

Comment:
It's a good practice for each layer to have its own model that it works with. In the domain layer, you'd have `Movie` model that's used in use cases for example. In the api layer, you'd have `ApiMovie` model that used in the api layer. UI layer would then have `MovieViewState`, etc. 
Since this would be the domain layer then, you shouldn't use anything that's Android specific here, like the `MutableState` for example.